lib/open-uri-cached-archive.rb

```diff
@@ -1,5 +1,43 @@
 require 'open-uri-cached-archive/version'
+require 'yaml'
+require 'stringio'
+require 'open-uri'
 
 class OpenUriCachedArchive
-  # Your code goes here...
+  class CachedResponse
```

Comment:
Ah yep, I'm very guilty of not doing that! Moved in https://github.com/everypolitician/open-uri-cached-archive/pull/3/commits/46d6f95bbe4b1f05f7d87287aa0070bc5422548a